packages/ui/forms/recover-account/index.tsx

```diff
@@ -164,10 +164,10 @@ class RecoverAccountForm extends Component<Props, State> {
                 onBlur={(): void => this.checkMnemonicLength()}
               />
             </View>
-            { mnemonicLengthError && <Text style={style.warningText}>{mnemonicLengthError}</Text> }
-            <Button round fat style={style.submitButton} onPress={() => this.setPIN()} text={recoverAccount} />
-            <Button alternate small arrow style={style.submitButton} onPress={() => this.cancel()} text={cancel} />
           </KeyboardAvoidingView>
+          { mnemonicLengthError && <Text style={style.warningText}>{mnemonicLengthError}</Text> }
+          <Button round fat style={style.submitButton} onPress={() => this.setPIN()} text={recoverAccount} />
```

Comment:
Avoid inline functions as callback props; they force a render of that component every time this one renders, because the props, technically, have changed.